### PR TITLE
Inprogress

### PR DESCRIPTION
**Pull request by ellie-r**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -0,0 +1,675 @@
...
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#Projects">Projects</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#contact">Contact Me</a>
+                    </li>
+                </ul>
+                <div class="cvLink">
+                    <a href="assets/CV-v2.pdf" target="_blank">
```
Comment (jennymegan) [inline app/index.html:38]: idk how to change it but when you open the CV in the top of the tab it says "blue and grey software developer ..." rather  than "Ellie R CV" or anything

File: app/index.html [hunk 2]
```diff
@@ -0,0 +1,675 @@
...
+                        <button type="button" class="btn">View CV</button>
+                    </a>
+                </div>
+            </div>
+
+        </nav>
+        <div class="fixed-top">
+            <div class="toggle">
+                <input type="checkbox" id="toggle">
```
Comment (jennymegan) [inline app/index.html:47]: this is FABULOUS and i love it

File: app/index.html [hunk 3]
```diff
@@ -0,0 +1,675 @@
...
+                            cy="408.62"
+                            r="9.713" />
+                    <circle
+                            id="XMLID_45_"
+                            style="fill:#65706C;"
+                            cx="244.986"
+                            cy="328.079"
+                            r="27.606" />
+</svg>
+                <i class="fas fa-angle-down fa-5x moreContent"></i>
```
Comment (jennymegan) [inline app/index.html]: this doesn't "jump" down the page when clicked - idk if it's supposed to?

File: app/index.html [hunk 4]
```diff
@@ -0,0 +1,675 @@
...
+                    I graduated from the University of Exeter in 2019 with a 2:1 in Maths.
+                    After graduating I worked for a while as a Junior Data Analyst where I decided that solving problems
+                    with code was the favourite part of my day and I wanted to pursue it as a career!
+                </p>
+                <p>
+                    I decided to join iO Academy (prev. Mayden Academy),
+                    a 16 week intensive course, to learn all the skills required for the industry.
+                </p>
+                <p>
+                    I like reading sci-fi and fantasy books, binging Netlix shows and playing with my two miniature dachshunds.
```
Comment (jennymegan) [inline app/index.html]: typo in Netflix

File: app/index.html [hunk 5]
```diff
@@ -0,0 +1,675 @@
...
+                </p>
+            </div>
+            <div class="col-md-3">
+                <img src="assets/IMG_7406.jpeg" alt="ellie at graduation">
```
Comment (jennymegan) [inline app/index.html:521]: greatly enjoy this photo

File: app/index.html [hunk 6]
```diff
@@ -0,0 +1,675 @@
...
+                <div class="col-sm-12 col-lg-6 pt-5 projectInfo text-center">
+                    <h3>E Commerce Store</h3>
+                    <div class="projectDescription">
+                        <p>
+                            Built an e-commerce store web app with object oriented PHP that implements the SOLID principles. Had to extract data
+                            from an API and import into a MySQL database. Worked as a scrum team of 8 to
+                            plan the architecture and estimate tasks.
+                        </p>
+                        <ul class="projectTechList">
+                            <li>PHP</li>
```
Comment (jennymegan) [inline app/index.html:544]: does also use mySQL / html/css? can see why you haven't put them but they are on the wine one below

File: app/index.html [hunk 7]
```diff
@@ -0,0 +1,675 @@
...
+                            <li>Bootstrap</li>
+                        </ul>
+                        <div class="social-menu">
+                            <a class="social-button github" href="https://github.com/Mayden-Academy/2020-sept-ecommerce" target="_blank">
+                                <i class="fab fa-github"></i>
+                            </a>
+                            <a class="social-button openProject" href="https://dev.io-academy.uk/projects/2020-sept/2020-sept-ecommerce/index.php" target="_blank">
```
Comment (jennymegan) [inline app/index.html:554]: you can "fork" the repo so this links to an ellie-r version on github

File: app/scss/style.scss [hunk 8]
```diff
@@ -1,5 +1,324 @@
...
+  }
+}
+
 body {
-  display: block;
-  color: red;
-  border: 1px;
+  background-color: var(--background);
+  color: var(--text);
+  font-family: 'Titillium Web', sans-serif;
```
Comment (jennymegan) [inline app/scss/style.scss:19]: appreciate the name of this font.

File: app/index.html (hunk 3 above)
Comment (ellie-r) [reply app/index.html]: it wasn't supposed to but i think it's a good idea i shall add it in! :)

**Pull request merged**